docs/api.md

```diff
@@ -719,11 +727,56 @@ The default browser context is the only non-incognito browser context.
 
 Creates a new page in the browser context.
 
+
+#### browserContext.overridePermissions(origin, permissions)
+- `origin` <[string]> An [origin] to grant permissions to.
+- `permissions` <[Array]<[string]>> An array of permissions to grant. All permissions that are not listed here will be automatically denied. Permission might be one of the following values:
+    - `'geolocation'`
+    - `'midi'`
+    - `'midi-sysex'` (system-exclusive midi)
+    - `'notifications'`
+    - `'push'`
+    - `'camera'`
+    - `'microphone'`
+    - `'background-sync'`
+    - `'ambient-light-sensor'`
+    - `'accelerometer'`
+    - `'gyroscope'`
+    - `'magnetometer'`
+    - `'accessibility-events'`
+    - `'clipboard-read'`
+    - `'clipboard-write'`
+    - `'payment-handler'`
+    - `'flash'` (chrome-specific permission)
+- returns: <[Promise]>
+
+
+```js
+const context = browser.defaultBrowserContext();
+await context.overridePermissions('https://html5demos.com', ['geolocation']);
+```
+
+> **NOTE** Permission management is scoped to browser context and is done per origins rather then
```

Comment:
I tried to fix the wording of this. But instead just remove this note. The api makes it clear how it works, and nobody needs to know why.

Reply:
Done.